apps/web/src/client/components/html-content-editor/html-content-editor.js

```diff
@@ -24,7 +24,18 @@ function initHtmlContentEditor() {
 			initialEditType: 'wysiwyg',
 			usageStatistics: false,
 			hideModeSwitch: true,
-			toolbarItems: [['bold', 'link', 'ul']]
+			toolbarItems: [['bold', 'link', 'ul']],
+			events: {
+				keydown(_, ev) {
```

Comment:
Seems reasonable, I think this is an edge case, my only question is if the check on each key press slows down the entry in any way, doubt it would.

Reply:
Good thought; the library itself checks every key press for many things (including inserting spaces if you press tab). So I think an extra listener will be negligible.